chromium_src/chrome/browser/ui/webui/settings/settings_secure_dns_handler.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/values.h"
+#include "brave/components/unstoppable_domains/buildflags/buildflags.h"
+
+#if BUILDFLAG(UNSTOPPABLE_DOMAINS_ENABLED)
+#include "brave/components/unstoppable_domains/constants.h"
+#endif
+
+namespace settings {
+namespace {
+std::unique_ptr<base::DictionaryValue> CreateSecureDnsSettingDict();
+}  // namespace
+}  // namespace settings
+
+#include "../../../../../../../chrome/browser/ui/webui/settings/settings_secure_dns_handler.cc"
+
+namespace settings {
+
+namespace {
+
+// Hide Unstoppable Domains resolver in the custom provider list in settings
+// because it will be used for name resolution only for TLDs from Unstoppable
+// Domains, instaed of a global DoH settings.
```

Comment:
instEAd

Reply:
Fixed in 5f9dc0f3cafa00174ff3872a74e14d1c2b15d561